sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/sqlstore/SQLLibraryDAO.java

```diff
@@ -398,7 +398,7 @@ public long save(Library library) throws IOException {
 
           String name = namingScheme.generateNameFor(library);
           library.setName(name);
-          DbUtils.validateNameOrThrow(library, namingScheme);
+          NamingScheme.validateNameOrThrow(library, namingScheme);
```

Comment:
Also, why remove the `DbUtils.validateNameOrThrow` helper method instead of using it here and removing `NamingSchemeAware` from the Store interface? (Are you thinking there'll be a case where we want to autowire different naming schemes into different DAOs?)